tests/integration/security/reachability_test.go

```diff
@@ -60,8 +60,8 @@ func TestReachability(t *testing.T) {
 					ConfigFile: "beta-mtls-permissive.yaml",
 					Namespace:  systemNM,
 					Include: func(src echo.Instance, opts echo.CallOptions) bool {
-						// Exclude calls to the naked app.
-						return opts.Target != rctx.Naked
+						// Exclude calls to the naked app and from naked->VM.
```

Comment:
nit: would be more useful to comment why we are doing this, not what we are doing - the comment here is redundant with the code

Reply:
updated!